Sources/eppo/EppoClient.swift

```diff
@@ -225,11 +260,11 @@ public class EppoClient {
         subjectAttributes: SubjectAttributes,
         expectedVariationType: UFC_VariationType) throws -> FlagEvaluation?
     {
-        if (self.sdkKey.count == 0) {
+        if (isOnlineMode && self.sdkKey.count == 0) {
```

Comment:
Note to self: This behavior needs a test

Reply:
I think it's fine to request an sdk key even if client is initialized from json. We want them to hold of a valid SDK key, and we might use it for something else in the future.